loky/backend/context.py

```diff
@@ -52,13 +52,20 @@ def assert_spawning(obj):
             )
 
     def get_context(method="loky"):
-        return LokyContext()
+        if method == "loky":
+            return LokyContext()
+        elif method == "loky_no_main":
+            if sys.platform == "win32":
+                return LokyContext()
```

Comment:
Instead of cloning the code can just derive the from the class and override some methods to disable the `init_main` behavior, no?